src/test/java/uk/gov/pay/ledger/event/EventIntegrationTest.java

```diff
@@ -16,19 +16,20 @@
 @Ignore
 public class EventIntegrationTest {
     @ClassRule
-    public static AppWithPostgresRule rule = new AppWithPostgresRule();
+    public static AppWithPostgresAndSqsRule rule = new AppWithPostgresAndSqsRule();
```

Comment:
We should move to `DropwizardJUnitRunner` so we can have one postgres/sqs container for all tests. But we can look at it separately.

Reply:
The longer the PR is hanging the more conflicts appear/more changes are required.
Moving to `DropwizardJUnitRunner` should not be part of this PR. It's too big already.